src/sentry/features/__init__.py

```diff
@@ -81,6 +81,7 @@
 default_manager.add("organizations:auto-enable-codecov", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
 default_manager.add("organizations:cleanup-project-serializer", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:codecov-commit-sha-from-git-blame", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+default_manager.add("organizations:crons-disable-new-projects", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
```

Comment:
You can add this to getsentry only actually

Reply:
oooh that is true, lemme fix

Reply:
https://github.com/getsentry/getsentry/blob/747f01d7516e40465c42c212bd1fca7fdc45bdd5/getsentry/features.py#L1891-L1912

Reply:
updated here: https://github.com/getsentry/getsentry/pull/12460